### ISSUE TITLE
(viaf88890045.003.xml) Smith bio broken entries

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/canonical-pdlrefwk#36]
user report IR20231001
The three entries for Stratocles seem to be missing from Smith's Dictionary of Greek and Roman Biography:

http://www.perseus.tufts.edu/hopper/invalidquery.jsp?doc=Perseus:text:1999.04.0104:alphabetic%20letter=S:entry%20group=27:entry=stra
http://www.perseus.tufts.edu/hopper/invalidquery.jsp?doc=Perseus:text:1999.04.0104:alphabetic%20letter=S:entry%20group=27:entry=stra
http://www.perseus.tufts.edu/hopper/invalidquery.jsp?doc=Perseus:text:1999.04.0104:entry=stra

**Comment by AlisonBabeu** [comment]
hi @lcerrato I did find them in the XML text itself though. They just aren't displaying!

**Comment by lcerrato** [comment]
@AlisonBabeu  right, I cannot tell if this is a code issue or an xml issue. It looks to be the links are not resolving correctly, as these do show in a search. But without any way to test, it could be a structural problem with this entry group.

**Comment by lcerrato** [comment]
On 3/11/15 9:50 AM, ... previous report
> In the A Dictionary of Greek and Roman biography and mythology I can’t visualize the entries between [Stratius](http://www.perseus.tufts.edu/hopper/text?doc=stratius-bio-5&fromdoc=Perseus%3Atext%3A1999.04.0104) and [Stratolas](http://www.perseus.tufts.edu/hopper/text?doc=stratolas-bio-1&fromdoc=Perseus%3Atext%3A1999.04.0104) and between [Autesion](http://www.perseus.tufts.edu/hopper/text?doc=autesion-bio-1&fromdoc=Perseus%3Atext%3A1999.04.0104) and [Autolaus](http://www.perseus.tufts.edu/hopper/text?doc=autolaus-bio-1&fromdoc=Perseus%3Atext%3A1999.04.0104).

**Comment by lcerrato** [comment]
Yes, they are there, but that doesn't mean the xml structure is correct. Sometimes when entries do not display in large reference works, there is an underlying xml error. This is pretty common in the lexica.

**Comment by lcerrato** [comment]
I think the text itself may be ok? Very hard to tell.
